### DISCUSSION TITLE
Can I trigger a background task manually?

### DISCUSSION THREAD

**Discussion by HengzheLi** [OrchardCMS/OrchardCore#9977] [Q&A]
I see Disable etc buttons in the BackgroundTask UI. Can I trigger a specified task manually through the UI?

**Comment by Skrypt** [comment]
Background tasks use a CRON to execute periodically so you can override this CRON value and set it to trigger every minutes. Then you can start/pause it from the background task list in the admin UI. There's nothing to trigger it instantly otherwise.
